### PR TITLE
feat: nft view page image and container improvements

### PR DESCRIPTION
**Pull request by domw30**
[Associated Notion Card](https://www.notion.so/zerotech/4985d8ec0c2f4eb4860678eec483e6f6?v=2de07237786c4d62830797da093e6a04&p=12be68402f3c45f4b6e6da60183298ce&pm=s)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
- feature / improvement

### CODE REVIEW

**Review by colbr — COMMENTED** [review]
I'm liking the refactoring here! I have a suggestion which will make this task a lot easier.

Instead of absolute positioning the `NFTBannerContainer` background, you could move the zApp content wrapper to each page. This means inside the `NFT` page, you could move the `NFTBannerContainer` to outside of the zApp content wrapper, and thus escape the existing padding/positioning.

It would look something like:
Untitled-2022-11-21-2212

**Comment by domw30** [comment]
> I'm liking the refactoring here! I have a suggestion which will make this task a lot easier.
> 
> Instead of absolute positioning the `NFTBannerContainer` background, you could move the zApp content wrapper to each page. This means inside the `NFT` page, you could move the `NFTBannerContainer` to outside of the zApp content wrapper, and thus escape the existing padding/positioning.
> 
> It would look something like: Untitled-2022-11-21-2212

Looks great, thanks for the suggestion!

File: src/features/view-nft/NFTBannerContainer/NFTBannerContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	const { data: metadata } = useDomainMetadata(domainId);
+
+	const altTemplate = `${metadata?.title ?? zna} nft `;
+
+	const bannerAlt = altTemplate + ` banner`;
+	const bannerSrc = metadata?.image_full ?? metadata?.image;
+
+	const { data: prominentColor } = useColorPallette(bannerSrc);
+
+	const bannerBackground = {
```
Comment (domw30) [inline src/features/view-nft/NFTBannerContainer/NFTBannerContainer.tsx:27]: Would there be a better way to pass the colour to module.scss file?

File: src/features/view-nft/NFTDetailsCard/ui/CTAContainer/CTAContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export interface CTAContainerProps {}
+
+export const CTAContainer: FC<CTAContainerProps> = ({}) => {
+	return (
+		<div className={styles.Container}>
+			<div className={styles.Content}>
+				<span className={styles.Label}>{'Buy Now (WILD)'}</span>
+				<div className={styles.Values}>
+					<span className={styles.TokenValue}>{'3,000,000.00'}</span>
```
Comment (domw30) [inline src/features/view-nft/NFTDetailsCard/ui/CTAContainer/CTAContainer.tsx:15]: Hard coded values. The data will be added in a follow up PR. Aiming to separate image alignment task and updates to NFT CTA's

**Review by colbr — APPROVED** [review]

**Pull request merged**